dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/zookeeper/ZookeeperClient.java

```diff
@@ -33,7 +33,7 @@ public interface ZookeeperClient {
      * @param path path to ZNode
      * @param ephemeral specify create mode of ZNode creation. true - EPHEMERAL, false - PERSISTENT.
      */
-    void create(String path, boolean ephemeral);
+    void create(String path, boolean ephemeral, boolean faultTolarance);
```

Comment:
Description of the addition of the `faultTolarance` parameter, other LGTM

Reply:
done